deployment/terraform/create.go

```diff
@@ -488,6 +491,19 @@ func (t *Terraform) setupAppServer(extAgent *ssh.ExtAgent, ip, siteURL, serviceF
 		}
 	}
 
+	if t.config.EnableNetPeekMetrics {
+		// Stop netpeek service in case it's already deployed
+		cmd = "sudo service netpeek stop"
+		if out, err := sshc.RunCommand(cmd); err != nil {
+			return fmt.Errorf("error running ssh command %q, output: %q: %w", cmd, string(out), err)
+		}
```

Comment:
Normally a restart is enough right?

Reply:
This was a leftover from the previous approach that replaced the binary so required a stop. I'll simplify, thanks.